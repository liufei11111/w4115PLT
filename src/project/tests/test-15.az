Int main(Int argc, String argv)｛
Int a;
a = 5;
Boolean b1;
Boolean b2;
b1 = Ture;
b2 = Talse;

while (a > 0){
	a = a - 1;
}

while (b1 && b2)
	a = 10;
｝