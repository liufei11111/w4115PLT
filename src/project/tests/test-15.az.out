
public static int main(int argc, String argv)
{
try{int a=0;
a = 5;
boolean b1=false;
boolean b2=false;
b1 = ture;
b2 = space;
while (a > 0) {
  a = a - 1;
}
while (b1 && b2) a = 10;
}catch(Exception e){return -1;}}

